db/migrate/20220512033818_add_role_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string, default: 'user'
+  end
+end
```

Comment:
- [ ] Within this milestone you should add divice for users table. So far you have succesfully installed divice but did not generate migration file for users. 
- [ ] to do that kindly run , `rails g devise user `

then open migration file and uncomment 
necessary fields like confirmable etc. 

then run `db:migrate`

After, check your users table it should get new password, email and other new fields 

- [ ] Finally, please make sure that you can create a new user using `rails c`.
- [ ] You can check divice documentation and watch videos on Youtube if you get stuck or keep getting errors.